src/Notify/Notifier/AppleNewsNotifier.php

```diff
@@ -156,12 +156,42 @@ protected function updateArticle(Message $notification, Message $app, Message $a
         }
 
         $latestRevision = $this->getLatestRevision($notification);
-        if (null === $latestRevision || $article->get('apple_news_revision') === $latestRevision) {
+        if (null !== $latestRevision && $metadata['revision'] !== $latestRevision) {
+            $metadata['revision'] = $latestRevision;
+            $result = $this->api->updateArticle((string)$article->get('apple_news_id'), $document, $metadata);
+        }
+
+        if ($result['ok']) {
+            return $result;
+        }
+
+        $getArticleResult = $this->api->getArticle((string)$article->get('apple_news_id'));
+        if (!$getArticleResult['ok']) {
+            $this->sendMessageToSlack($article, $notification);
```

Comment:
@tbone1020 instead of adding specific "sendMessageToSlack" feature, simply return result as normal and add your own `tmz:notify:node:apple-news-notification.notification-failed` event listener to your SlackWatcher and handle there.  sending to slack is a site level handling